docs/apworld_dev_faq.md

```diff
@@ -66,3 +66,21 @@ The reason entrance access rules using `location.can_reach` and `entrance.can_re
 We recognize it can feel like a trap since it will not alert you when you are missing an indirect condition, and that some games have very complex access rules.
 As of [PR #3682 (Core: Region handling customization)](https://github.com/ArchipelagoMW/Archipelago/pull/3682) being merged, it is possible for a world to opt out of indirect conditions entirely, instead using the system of checking each entrance whenever a region has been reached, although this does come with a performance cost.
 Opting out of using indirect conditions should only be used by games that *really* need it. For most games, it should be reasonable to know all entrance &rarr; region dependencies, making indirect conditions preferred because they are much faster.
+
+---
+
+### I uploaded the generated output of my world to the webhost and webhost is erroring on corrupted multidata
+
+The error `Count not load multidata. File may be corrupted or incompatible.` on uploading a locally generated file is caused because output multidata is the pickled representation of the data that the server needs to know. It may come
+with a descriptive issue like `(No module named 'worlds.myworld')` or `(global 'worlds.myworld.names.ItemNames' is forbidden)`
```

Comment:
```suggestion
The error `Could not load multidata. File may be corrupted or incompatible.` occurs when uploading a locally generated file where there is an issue with the multidata contained within it. It may come
with a descriptive issue like `(No module named 'worlds.myworld')` or `(global 'worlds.myworld.names.ItemNames' is forbidden)`
```